config/winston.js

```diff
@@ -13,12 +17,49 @@ const logger = createLogger({
 
 const developmentFormat = printf(info => `${info.timestamp} ${info.level}: ${info.message}`);
 
-if (config.env !== 'production') {
-    logger.format = combine(
+git.getLastCommit((err, commit) => {
+    const prodFormat = printf(info => JSON.stringify({
+        service: pjson.name,
+        logger: 'my_winston_logger',
+        hostname: os.hostname(),
+        level: info.level,
+        msg: info.message,
+        meta: {
+            service: {
+                commit: commit.shortHash,
+                commitMessage: commit.subject,
+                version: pjson.version,
+            },
+            logger: {
+                time: info.timestamp,
+                baz: 'bok',
+            },
+            event: {
```

Comment:
Even with the formater
```
logger.info("hello", {'foo': bar})
```

Would yield
```
{
  msg: 'hello'
  foo: 'bar'
}
```
Do you know how easy it'd be to have it go into "event"?
```

{
  msg: 'hello'
  event: {
     foo: 'bar'
  }
}
```

Reply:
Done.